test/test_prototype_transforms_consistency.py

```diff
@@ -840,3 +850,80 @@ def test_aa(self, inpt, interpolation):
         output = t(inpt)
 
         assert_equal(expected_output, output)
+
+
+# Import reference detection transforms here for consistency checks
+# torchvision/references/detection/transforms.py
+ref_det_filepath = Path(__file__).parent.parent / "references" / "detection" / "transforms.py"
+det_transforms = SourceFileLoader(ref_det_filepath.stem, ref_det_filepath.as_posix()).load_module()
```

Comment:
Why the `.as_posix()` here? Does that work on Windows? I'll guess CI will tell us. Did you mean to convert into a string?

```suggestion
det_transforms = SourceFileLoader(ref_det_filepath.stem, str(ref_det_filepath)).load_module()
```

Reply:
yes, it will work on windows as well.